virt_lightning/shell.py

```diff
@@ -20,47 +22,75 @@
     "username": getpass.getuser(),
 }
 
+lang, encoding = locale.getdefaultlocale()
+
+if encoding and encoding == "UTF-8":
```

Comment:
I don't want to keep this block in this file. It should be moved either in a new module. e.g:

```python
import virt_lightning.symbol as symbol
print(symbol.ok)
print(symbol.ko)
```

Reply:
Yes, you true it will more useful.